lib/telemetry_metrics_statsd.ex

```diff
@@ -402,6 +402,18 @@ defmodule TelemetryMetricsStatsd do
     end
   end
 
+  def get_udp_worker(pool_id) do
+    case :ets.lookup(pool_id, :udp_worker) do
+      [] ->
+        Logger.error("Failed to publish metrics over UDP: no workers available.")
```

Comment:
It's a bit weird to see this error here. The function is just a worker locator; maybe the error should be logged not inside this function but in the calling function

Reply:
this is consistent with [this part](https://github.com/svsool/telemetry_metrics_statsd/blob/main/lib/telemetry_metrics_statsd.ex#L396), and doesn't change the end behavior in the end

Reply:
done for consistency with [get_udp](https://github.com/svsool/telemetry_metrics_statsd/blob/main/lib/telemetry_metrics_statsd.ex#L396), easy to move though to the caller in this case since used only once, so moved.